### ISSUE TITLE
Kinesis stream bug in China AWS

### ISSUE THREAD

**Issue by DanishKhakwani** [terraform-providers/terraform-provider-aws#3566]
### Terraform Version
Terraform v0.11.1
+ provider.aws v1.10.0

### Affected Resource(s)
- aws_kinesis_stream

### Terraform Configuration Files
```hcl
resource "aws_kinesis_stream" "kstream" {
  name             = "kstream"
  shard_count      = 64
  retention_period = 168

  shard_level_metrics = [
    "IncomingBytes",
    "OutgoingRecords",
    "IteratorAgeMilliseconds",
    "IncomingRecords",
    "ReadProvisionedThroughputExceeded",
    "WriteProvisionedThroughputExceeded",
    "OutgoingBytes"
  ]
}
```

When the above resource is applied, the following change keeps showing up on subsequent apply/plan.
```
  ~ aws_kinesis_stream.kstream
      encryption_type: "" => "NONE"
```

I have tried with both *encryption_type* defined and undefined (which should default to "NONE").

### Expected Behavior
What should have happened?
*No change*

### Actual Behavior
What actually happened?
*1 to change*

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply -target aws_kinesis_stream.kstream`
2. `terraform plan -target aws_kinesis_stream.kstream`

*NOTE*: I am only seeing this on AWS China Account. Other regions are fine.

**Comment by bflad** [comment]
Bug fix pull request submitted: #4575

**Comment by bflad** [comment]
The fix for this will release with v1.20.0 of the AWS provider, likely later today. 👍

**Comment by ryanmaclean** [comment]
Fantastic news!

**Comment by bflad** [comment]
This has been released in [version 1.20.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1200-may-23-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
